src/clj/game/cards/identities.clj

```diff
@@ -639,7 +639,9 @@
                                      :rfg))}}]}
 
    "Kate \"Mac\" McCaffrey: Digital Tinker"
-   {:events {:pre-install {:req (req (and (#{"Hardware" "Program"} (:type target))
+   {:effect (req (when (pos? (event-count state side :runner-install))
+                   (swap! state assoc-in [:per-turn (:cid card)] true)))
+    :events {:pre-install {:req (req (and (#{"Hardware" "Program"} (:type target))
```

Comment:
This will clash with my fixes PR, and this effect is too restrictive. It will prevent Kate discount if you have installed anything before you Rebirth.

Reply:
Yeah, my bad for not pulling this out after talking with you yesterday. Removing this change and the pending change should revert that bit.